samples/nrf9160/modem_shell/src/uart/uart_shell.c

```diff
@@ -18,9 +18,11 @@
 #include "uart_shell.h"
 #include "uart.h"
 #include "mosh_print.h"
+#include "link.h"
 
 
 bool uart_disable_during_sleep_requested;
+extern bool msleep_notifications_subscribed;
```

Comment:
@jhirsi, should we have getter and setter functions for this in link_api.c or just keep it an extern variable?

Reply:
We have varying implementations for this kinds. Getter would be nicer, but I'm fine with this. Although, naming should be indicating the owner of the variable. For example this should be like:  "link_shell_msleep_notifications_enabled"

Reply:
I thought link module would not use extern too much and seems it's not providing other globals used by other modules via extern. It's using extern variables from other modules though.
The reason I thought we could avoid making extern variables in link module is because it has this link_api for other modules to use so it's clearly a module that provides services to other modules.

Reply:
That being said, I don't have very strong opinion either way. I've had thoughts on getting rid of externs in print module so that's the reason I commented this in the first place.

Reply:
Improved naming of this variable as well as uart_disable_during_sleep_requested which is similar but defined in uart_shell.c instead and exported in link.c